pkg/plank/OWNERS

```diff
@@ -7,4 +7,4 @@ approvers:
 emeritus_approvers:
 - chaodaiG
 labels:
- - area/prow/plank
+ - area/plank
```

Comment:
We have not migrated this label, this needs to be removed - tbh I'm not sure if we can just deprecate this, or if this code is `area/prowcm` now